drivers/mge-hid.c

```diff
@@ -1292,7 +1294,7 @@ static hid_info_t mge_hid2nut[] =
 	/* Refer to Note 1 (This point will need more clarification!)
 	{ "battery.charger.status", 0, 0, "UPS.BatterySystem.Charger.PresentStatus.Used", NULL, "%.0f", HU_FLAG_QUICK_POLL, eaton_abm_enabled_legacy_info }, */
 	/* This data is the actual ABM status information */
-	{ "battery.charger.status", 0, 0, "UPS.BatterySystem.Charger.Mode", NULL, "%.0f", HU_FLAG_QUICK_POLL, eaton_abm_status_info },
+	{ "battery.charger.status", 0, 0, "UPS.BatterySystem.Charger.Status", NULL, "%.0f", HU_FLAG_QUICK_POLL, eaton_abm_status_info },
```

Comment:
to be tested, may be avail on some units, not others. May need both Mode and Status

Unified charger info,bitmap with :
Status on bit 0-3:
 0x0: Off
 0x1: Charging
 0x2: Floating
 0x3: On, not charging
 0x4: Discharging
 0x5-0xF : Reserved
Mode on bit 4-6:
 0: CC
 1: ABM
 2-7 : Reserved
Error on bit 7:
 0: Good
 1: In error

Reply:
Also ABM table commented needs to be changes?